interop/client/client.go

```diff
@@ -30,6 +30,7 @@ import (
 
 	"google.golang.org/grpc"
 	_ "google.golang.org/grpc/balancer/grpclb"
+	_ "google.golang.org/grpc/balancer/rls"
```

Comment:
While you are here could you please group the blank imports as a separate group and add comments for why we import those packages.

This would be something like:

```golang
	_ "google.golang.org/grpc/balancer/grpclb" // Register the grpclb load balancing policy.
	_ "google.golang.org/grpc/balancer/rls" // Register the RLS load balancing policy.
	_ "google.golang.org/grpc/xds/googledirectpath" // Register xDS resolver required for c2p directpath.
```

Reply:
Ping on this

Reply:
Fixed. Sorry I missed this comment.